pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -201,6 +201,8 @@ stdenv.mkDerivation rec {
     for f in $dev/bin/gtk-encode-symbolic-svg; do
       wrapProgram $f --prefix XDG_DATA_DIRS : "${shared-mime-info}/share"
     done
+
+    GTK_PATH="''${out:?}/lib/gtk-3.0/3.0.0/immodules/" ''${dev:?}/bin/gtk-query-immodules-3.0 > "''${out:?}/lib/gtk-3.0/3.0.0/immodules.cache"
```

Comment:
Looking at `git checkout ca3f089a836 && nix-build -A gtk3 --system x86_64-darwin`, we probably want to do that on Darwin as well.

Reply:
like so? [41e9b1d](https://github.com/NixOS/nixpkgs/pull/172616/commits/41e9b1d60418c1c021eb306200d37efe22e07859)

Reply:
`++` is for joining lists, strings need `+`. But otherwise looks good.

Reply:
Huh, you're right. Thank you!